tests/db_test.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+import sqlalchemy
+from sqlalchemy import create_engine
+from sqlalchemy_utils import database_exists
+
+
+db_uri = 'postgresql://postgres:edison@127.0.0.1/edison'
+
+@pytest.fixture
+def engine():
+    return create_engine(db_uri)
+
+def test_db_exist():
+    assert database_exists(db_uri) is True
+
+def test_db_connection(engine):
+    assert engine.connect() is not None
```

Comment:
@simzacks I'm not sure with this db connection check, I tried to find something more correct but without success

Reply:
@ahinoamta what isn't working with this code?
IIUC, you are:
* connecting to the database, which verifies that it exists in the location/port that you are expecting it to be
* verifying that the expected tables exist.

Reply:
This code is working, but don't you think the check needs to be more specific and not just `is not None`?

Reply:
2 questions
1) Is there ever a time that you call  database_exists and it returns true, but engine.connect would fail?
2) If you could not connect to the engine, what would engine.connect return?

Reply:
@simzacks 
1. I think there is no such case, I googled it and I didn't find any other case that can cause failure to `engine.connect`. I did find that the engine provides the `Connection` from the `connection pool` but I'm not sure it can cause connection failures (assuming the DB is up).

2. If the database doesn't exist, so I won't be able to connect to the engine, it will throw an exception- 'sqlalchemy.exc.OperationalError'.